### PR TITLE
Feature/create main component

### PR DESCRIPTION
**Pull request by sleepergoose**
Create a main component
Combine left-side menu and main page (with Khrapal Maksym)

### CODE REVIEW

File: frontend/src/app/moduls/main/cards/calm-rhythms-card/calm-rhythms-card.component.sass [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.cr-card
+    height: 228px
+    width: 234px
+    margin-right: 25px 
+    border-radius: 4px
+    background:#282F37
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/moduls/main/cards/calm-rhythms-card/calm-rhythms-card.component.sass]: Add space

File: frontend/src/app/moduls/main/cards/calm-rhythms-card/calm-rhythms-card.component.sass [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                opacity: 0.7
+        .play-icon
+            position: absolute
+            top: 28px
+            right: 27px
+            height: 28px
+            width: 28px
+            border: none
+            border-radius: 2px
+            background: linear-gradient(180deg, #FD8427 0%, #FD6831 100%)
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/moduls/main/cards/calm-rhythms-card/calm-rhythms-card.component.sass]: This gradient is duplicated five times in this pull request. Use variables

File: frontend/src/app/moduls/main/main-home/main-home.component.ts [hunk 3]
```diff
@@ -1,10 +1,111 @@
...
+  accordionAnimation = () => {
+    document.getElementsByClassName('accordion-information')[0]?.animate([
+      { opacity: '0' },
+      { opacity: '1' }
+    ], {
+      duration: this._animationDuration
+    });
+  };
+
+  scroll = (id: string): void => {
```
Comment (NikitaKrasnovSW) [inline frontend/src/app/moduls/main/main-home/main-home.component.ts]: There's no need to explicitly write return type. TS compiler understands the type in 99% of cases

**Review by NikitaKrasnovSW — CHANGES_REQUESTED** [review]

**Comment by sleepergoose** [comment]
Fixed the review notes and used variables wherever possible

**Comment by sleepergoose** [comment]
I'm really sorry! Since I forgot to make the profile menu a separate component, I included it in this pull request.

**Review by NikitaKrasnovSW — APPROVED** [review]

**Pull request merged**